### ISSUE TITLE
sorting the keys before writing documents to file

### ISSUE THREAD

**Issue by chaoranxie** [willhlaw/node-firestore-backup-restore#23]
is there any way to sort the keys before writing the documents to file? Basically, I am committing the backup files to version control but every time I do a new backup it shows all the documents are changed.

From what I can tell, following code is all that's needed
```
var stringify = require('json-stable-stringify');
fileContents = stringify(documentBackup, { space: 2 });
```

instead of
```
fileContents = JSON.stringify(documentBackup, null, 2);
```

**Comment by willhlaw** [comment]
Good idea. And that library is by @substack and looks good to me. I'll accept a PR for this.

**Comment by willhlaw** [comment]
@chaoranxie Thanks again for the PR. It was accepted and the new --stable option was published as 1.3.0.
